client/src/pages/Login.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Header from "../components/Header";
+
+class Login extends React.Component {
+  render() {
+    return (
+      <div>
+        {/* <Header /> */}
+        <div className="Title">The routes are set</div>
```

Comment:
This should be a lowercase "title" className to be consistent

Reply:
got it!